packages/tldraw/src/lib/shapes/shared/RichTextLabel.tsx

```diff
@@ -210,6 +216,7 @@ export function RichTextSVG({
 	wrap,
 	labelColor,
 	padding,
+	showTextOutline = true,
```

Comment:
does `SvgTextLabel.tsx` need this logic as well?

Reply:
The `stroke` property controls the outline